### ISSUE TITLE
Fallback font

### ISSUE THREAD

**Issue by marchellodev** [material-foundation/google-fonts-flutter#116]
It would be great to be able to specify a fallback font. So if a character doesn't exist in the main font, the glyph from the fallback one is used. It would make the process of adding new languages to the app a bit easier.

**Comment by gabrc52** [comment]
This would be nice, and at the moment this is a breaker for apps that include other languages.

> It would make the process of adding new languages to the app a bit easier.

@marchellodev Did you find a workaround?

**Comment by marchellodev** [comment]
@gabrc52 Yep, I store the name of the font in the localisation file, and then call `GoogleFonts.getFont(fontName)`
